### ISSUE TITLE
Square off edges in PoliciesModelledPopup

### ISSUE THREAD

**Issue by anth-volk** [PolicyEngine/policyengine-app#1730]
The edges in the below should be squared off. This is within the household calculator side of the app.

<img width="490" alt="Screen Shot 2024-05-08 at 7 53 01 PM" src="https://github.com/PolicyEngine/policyengine-app/assets/14987227/2746f023-1139-47d4-a0e2-ced8636f2a1e">

**Comment by LDraco-Sharif** [comment]
Hi @anth-volk, I was looking into your code and was also surprised why borders were not squared despite setting borderRadius to 0. Then I looked into the debugger.

The highlighted element here is the one you need to square off.
![image](https://github.com/PolicyEngine/policyengine-app/assets/33747246/a66c649d-eec6-435d-b246-60e285a8d8ca)

But this is the element where the style (or any custom class) gets applied.
![image](https://github.com/PolicyEngine/policyengine-app/assets/33747246/a740f34d-313a-461a-88b1-58d400321c10)

So, based on some forum posts, this is what I decided to do. I removed the style and added a new class name to the modal.
Then, I updated `App.css` to look for any element within that class with the class name `ant-modal-content` and set the border radius from there.

I have made a PR for this [here](https://github.com/PolicyEngine/policyengine-app/pull/1759). Let me know if it works on your end.

**Comment by anth-volk** [comment]
Thanks for this @LDraco-Sharif. Something we should put in our wiki (being built out as we speak) is that we typically prefer inline JS styling over CSS. We do have squared off borders on `src/modals/SignupModal.jsx`, so would you mind mimicking what's written there to implement this? We do appreciate your interest in contributing, just want to keep to the standards that were chosen earlier on.

**Comment by LDraco-Sharif** [comment]
@anth-volk, I tried using the method in `src/modals/SignupModal.jsx` but did not get too far. So, I went back and further checked documentation and found that Ant Design Modal does allow styling for `ant-modal-content`. 

Since this fix is unrelated to the previous fix, I have decided to close that PR and open a new one [here](https://github.com/PolicyEngine/policyengine-app/pull/1766). Let me know your thought as usual.
